akka-docs/src/main/paradox/stream/operators/Source/cycle.md

```diff
@@ -14,10 +14,10 @@ Stream iterator in cycled manner.
 
 ## Description
 
-Stream iterator in cycled manner. Internally new iterator is being created to cycle the one provided via argument meaning
-when original iterator runs out of elements process will start all over again from the beginning of the iterator
-provided by the evaluation of provided parameter. If method argument provides empty iterator stream will be terminated with
-exception.
+Stream iterator in cycled manner. Internally a new iterator is being created to cycle the one provided via argument meaning
+when original the iterator runs out of elements process will start all over again from the beginning of the iterator
+provided by the evaluation of provided parameter. If the method argument provides an empty iterator the stream will be 
```

Comment:
perhaps quote 'method'?

Reply:
I'm not sure I understand why the quoting is necessary.

Reply:
When I start reading this sentence and see "If the method...", I read 'method' as the English word 'method'. Then when I continue and read 'argument', I have to double-take and reinterpret 'method' as the name of the argument instead. Quoting removes that confusion. Not a big deal.

Reply:
```suggestion
provided by the evaluation of provided parameter. If the `method` argument provides an empty iterator the stream will be 
```

Reply:
Actually, with what @raboof said I think the better approach is to remove the word ' method'  altogether. It doesn't make any sense to say `If the method argument...` if we can say `If the argument...`. Of course it is an argument of a method.
```suggestion
provided by the evaluation of provided parameter. If the argument provides an empty iterator the stream will be 
```

Reply:
@raboof @dwijnand WDYT? (I'm adding a screenshot of the current, rendered page for extra content)

![screen shot 2018-12-04 at 14 58 36](https://user-images.githubusercontent.com/762126/49446756-2ecbc100-f7d5-11e8-81ba-41bd583e7bf6.png)